data_structures/src/superblock.rs

```diff
@@ -127,26 +142,19 @@ impl SuperBlockState {
     /// is in the ARS.
     /// Returns true, false, or unknown
     fn is_valid(&self, sbv: &SuperBlockVote) -> Option<bool> {
-        match self.current_superblock_index {
-            // We do not know the current index, we cannot know if the vote is valid
-            None => None,
-            // If the index is the same as the current one, the vote is valid if it is signed by a
-            // member of the ARS
-            Some(x) if x == sbv.superblock_index => self
-                .previous_ars_identities
+        if self.current_superblock_index == sbv.superblock_index {
+            self.current_signing_committee
```

Comment:
Hey, don't remove the comments
```suggestion
            // If the index is the same as the current one, the vote is valid if it is signed by a
            // member of the current signing committee
            self.current_signing_committee
```

Reply:
Is he?